src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const RENT_PRICE = 40;
+  const DAY_7 = 7;
+  const DAY_3 = 3;
+  const DISCOUNT_AFTER_DAY_7 = 50;
+  const DISCOUNT_AFTER_DAY_3 = 20;
+  let rent = RENT_PRICE * days;
```

Comment:
You've used `let` keyword to declare `rent` variable. But `let` allows the variable to be reassigned, which is not needed in this case. You can use `const` instead. This will make your code safer and easier to understand because it guarantees that the variable won't be reassigned later in the code. You can calculate the `rent` directly in the `if` conditions and return it. This will also simplify your code, because you won't need to subtract discounts from `rent`, you can just return `rent - discount`.